<library path="nav2_astar_planner">
	<class type="nav2_astar_planner::AstarPlanner"
	base_class_type="nav2_core::GlobalPlanner">
	  <description></description>
	</class>
</library>
